assets/js/opensuse-theme.js

```diff
@@ -186,6 +186,17 @@ function osMoreInformation(os) {
     $('.back-to-main-page').on('click', function() {
       backToMainPageOs($osSelected)
     })
+    
+    $('#home, #opensuseLogo').on('click', function() {
+        scrolled = $(window).scrollTop();
```

Comment:
can you check your indentation here ? It looks like you are using more than 2 spaces and we really want to have a clean indentation in all documents :)